jest/setup.ts

```diff
@@ -0,0 +1,11 @@
+import "@testing-library/jest-dom";
+
+type MockDocument = {
+  fonts: {
+    load: () => Promise<FontFace[]>;
+  };
+};
+
+(global.document as unknown as MockDocument).fonts = {
```

Comment:
Custom type otherwise you would get a typescript error as `fonts` is a read-only property.